rest_framework_inclusions/core.py

```diff
@@ -16,16 +16,23 @@ class Error(Exception):
 
 
 class InclusionLoader:
+    nested_inclusions_use_parent = True
+
     def __init__(self, allowed_paths):
         self.allowed_paths = allowed_paths
         self._seen = set()
 
+    def get_model_key(self, obj, *args, **kwargs):
+        return obj.__class__._meta.label
```

Comment:
```suggestion
    def get_model_key(self, obj, *args, **kwargs):
        return obj._meta.label
```
should have the same effect